frontend/packages/helm-plugin/console-extensions.json

```
@@ -119,6 +127,20 @@
       "icon": { "$codeRef": "helmUtils.helmCatalogIconSVG" }
     }
   },
+  {
+    "type": "dev-console.add/action",
+    "properties": {
+      "id": "project-helm-chart-repositories",
+      "groupId": "helm-chart-repositories",
+      "href": "/ns/:namespace/projecthelmchartrepositories/~new",
+      "label": "%helm-plugin~Helm Chart repositories%",
+      "description": "%helm-plugin~Add a Helm Chart Respository in the namespace%",
```

shouldn't this be ` Helm Chart repository` ? We use lowercase in the project helm chart form's helptext. cc: @debsmita1 
 
```suggestion
      "description": "%helm-plugin~Add a Helm Chart repository in the namespace%",
```